t/backup_expiration.t

```diff
@@ -160,18 +171,19 @@ describe "HTFeed::BackupExpiration" => sub {
       if ($old_versions[0]->{timestamp} > $old_versions[1]->{timestamp}) {
         ($newer, $older) = @old_versions;
       }
+      my $storage = prepare_storage($vars{storage_name}, new_random_timestamp());
```

Comment:
Shouldn't require a version < 6 months old -- perhaps consider in this case having 3 versions > 6 months old and ensuring all but the most recent of those gets deleted